tests/Common/TokensShareTest.php

```diff
@@ -23,5 +23,7 @@ public function testTokenShare()
     {
         $newToken = $this->tokens->createToken(new TokenCreateOptions());
         $this->tokens->shareToken($newToken['id'], 'test@devel.keboola.com', 'Hi');
+        // not to be marked as risky test
+        $this->assertTrue(true);
```

Comment:
$this->expectNotToPerformAssertions() ?

Nebo vymyslet nějakou věc, kterou můžeš assertnout na té response třeba...

Reply:
Ha, to jsem neznal, to by slo.

Reply:
jj, sry za nevyžádané review, ale přišlo mi to lepší teď než potom :)